docs/style-guide.md

```diff
@@ -4,10 +4,10 @@ layout: layouts/content.njk
 title: Content style guide
 permalink: /content-standards/style-guide/
 eleventyNavigation:
- key: Content style guide
- parent: Standards and principles
- order: 20
- excerpt: "The MoJ A to Z content style guide covering style, spelling and grammar conventions."
+  key: Content style guide
+  parent: Standards and principles
+  order: 20
+  excerpt: 'The MoJ A to Z content style guide covering style, spelling and grammar conventions.'
```

Comment:
Should we add [`singleQuote: false`](https://github.com/alphagov/govuk-frontend/blob/ed59f7f577907864550205b838a705fdf4147046/.prettierrc.js#L15C9-L15C27) for Markdown?

Reply:
👍 yep, this was my first thought too after running this.